Src/BootCamp.Chapter1/ArrayOperations.cs

```diff
@@ -63,8 +137,9 @@ public static int[] RemoveAt(int[] array, int index)
         /// <returns>A new array with element added at a given index. If an array is empty or null, returns new array with number in it.</returns>
         public static int[] InsertFirst(int[] array, int number)
```

Comment:
Reuse INsertAt

Reply:
didn't think about it , instead of create a new implementation each time better if you use what you got in hand